<?xml version="1.0"?>
<launch>
    <node name="stim300" pkg="stim" type="stim_pub" output="screen">
        <param name="device_id"  value="/dev/stim"/>
        <param name="topic_name" value="imu_data"/>
        <param name="frame_id" value="imulink" />
    </node>
</launch>
